app/src/main/java/comcmput301f16t01/github/carrier/MainActivity.java

```diff
@@ -156,6 +156,11 @@ public boolean onOptionsItemSelected(MenuItem item) {
             startActivity(intent);
         }
 
+        if (id == R.id.action_help) {
+            Intent intent = new Intent(this, HelpActivity.class);
```

Comment:
gah. Its not working... I will have to work on this at the airport... ugh... this may be late ish